src/core/layout/navbar/responsive-drawer/desktop/desktop.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Hidden } from '@material-ui/core'
+
+import { Login } from 'login'
+import { AppBar } from 'core/layout/navbar/appbar'
+import { Header } from 'core/layout/navbar/header'
+
+export const DesktopNavbar = () => <Hidden xsDown>
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/128638373-709599f8-811c-4639-88a6-e54d64537093.png)
Let's rename this file to `index.jsx`

Reply:
![image](https://user-images.githubusercontent.com/23569938/128638447-baed82fb-f97e-4bfb-b237-bba804a4bcf9.png)
Also, take a look at this. See how we use the `<Header/>` component here, but it is not only higher up on the directory tree, but also down another branch! Maybe we can remove the `responsive-drawer` directory (whose naming I didn't find proper anyways) and just have our layouts flat. Or we can create a separate `common` directory for the components used by both layouts.

Maybe something like this will do:
![image](https://user-images.githubusercontent.com/23569938/128638582-8fa664f2-ea5f-4900-897f-3d2fd4bf42cf.png)


Let's also rename the `navigation` directory to links.

Reply:
Done